test/e2e/snaps/test-snap-txinsights-v2.spec.js

```diff
@@ -3,166 +3,283 @@ const {
+          await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+          // wait for and click confirm and wait for window to close
+          await driver.waitForSelector({
+            text: 'Connect',
+            tag: 'button',
+          });
+          await driver.clickElementAndWaitForWindowToClose({
+            text: 'Connect',
+            tag: 'button',
+          });
+          // switch to test-snaps page and send tx
+          await driver.switchToWindowWithTitle(WINDOW_TITLES.TestSnaps);
+          await driver.clickElement('#sendInsights');
+          // delay added for rendering (deflake)
+          await driver.delay(2000);
+          // switch back to MetaMask window and switch to tx insights pane
+          await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+          // find confirm button
+          await driver.findClickableElement({
+            text: 'Confirm',
+            tag: 'button',
+          });
+          // wait for and click insights snap tab
+          await driver.waitForSelector({
+            text: 'Insights Example Snap',
+            tag: 'button',
+          });
+          await driver.clickElement({
+            text: 'Insights Example Snap',
+            tag: 'button',
+          });
+          // check that txinsightstest tab contains the right info
+          await driver.waitForSelector({
+            css: '.snap-ui-renderer__content',
+            text: 'ERC-20',
+          });
+          // click confirm to continue
+          await driver.clickElement({
+            text: 'Confirm',
+            tag: 'button',
+          });
+          // check for warning from txinsights
+          await driver.waitForSelector({
+            css: '.snap-delineator__header__text',
+            text: 'Warning from Insights Example Snap',
+          });
+          // check info in warning
+          await driver.waitForSelector({
+            css: '.snap-ui-renderer__text',
+            text: 'ERC-20',
+          });
+          // click the warning confirm checkbox
+          await driver.clickElement('.mm-checkbox__input');
+          // click confirm button to send transaction
+          await driver.clickElement({
+            css: '.mm-box--color-error-inverse',
+            text: 'Confirm',
+            tag: 'button',
+          });
+          // switch back to MetaMask tab
+          await driver.switchToWindowWithTitle(
+            WINDOW_TITLES.ExtensionInFullScreenView,
+          );
+          // switch to activity pane
+          await driver.clickElement({
+            tag: 'button',
+            text: 'Activity',
+          });
+          // wait for transaction confirmation
+          await driver.waitForSelector({
+            css: '.transaction-status-label',
+            text: 'Confirmed',
+          });
+        },
+      );
+    });
+  });
+
+  describe('Redesigned confirmation screens', function () {
+    it('tests tx insights v2 functionality', async function () {
```

Comment:
Transaction insights v2 is meant to show the modal, but that hasn't been ported to the redesigned confirmations. I don't think this test is testing anything anymore

Reply:
I noticed the modal wasn't there while migrating the test, but should we keep the test as a placeholder, and assert the modal existence once it's ported?